algorithmic_efficiency/workloads/librispeech_conformer/librispeech_jax/spectrum_augmenter.py

```diff
@@ -17,13 +17,13 @@ class SpecAug(nn.Module):
   This is an essential component in speech recognition models that helps achieve
   better word error rates.
   """
-  freq_mask_count: int = 1
-  freq_mask_max_bins: int = 15
-  time_mask_count: int = 1
-  time_mask_max_frames: int = 50
-  time_mask_max_ratio: float = 1.0
+  freq_mask_count: int = 2
+  freq_mask_max_bins: int = 27
+  time_mask_count: int = 10
+  time_mask_max_frames: int = 40
+  time_mask_max_ratio: float = 0.05
   time_masks_per_frame: float = 0.0
-  use_dynamic_time_mask_max_frames: bool = False
+  use_dynamic_time_mask_max_frames: bool = True
```

Comment:
These are now different from the internal values, is this intentional?

Reply:
internally we have default values and actual values we pass in while calling specaug layer from conformer / deepspeech models , I've just removed defaults and used the runtime values here to avoid confusion